fiddler/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+fiddler.accuracy,gauge,,percent,,"The accuracy of the model",0,fiddler,accuracy,
```

Comment:
You don't need the quotation marks around the descriptions here

Reply:
Removed the quotes.